<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginsWidget</class>
 <widget class="QWidget" name="PluginsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>332</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Plugins</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="sortButton">
       <property name="toolTip">
        <string>Sort the plugins using LOOT.</string>
       </property>
       <property name="whatsThis">
        <string>Sort the plugins using LOOT.</string>
       </property>
       <property name="text">
        <string>Sort</string>
       </property>
       <property name="icon">
        <iconset resource="../../../modorganizer/src/resources.qrc">
         <normaloff>:/MO/gui/sort</normaloff>:/MO/gui/sort</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="listOptionsBtn">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Open list options...</string>
       </property>
       <property name="whatsThis">
        <string>Refresh list. This is usually not necessary unless you modified data outside the program.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../modorganizer/src/resources.qrc">
         <normaloff>:/MO/gui/settings</normaloff>:/MO/gui/settings</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restoreButton">
       <property name="toolTip">
        <string>Restore a backup.</string>
       </property>
       <property name="whatsThis">
        <string>Restore a backup.</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../../modorganizer/src/resources.qrc">
         <normaloff>:/MO/gui/restore</normaloff>:/MO/gui/restore</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="toolTip">
        <string>Create a backup.</string>
       </property>
       <property name="whatsThis">
        <string>Create a backup.</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../../modorganizer/src/resources.qrc">
         <normaloff>:/MO/gui/backup</normaloff>:/MO/gui/backup</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="activePluginsLabel">
       <property name="text">
        <string>Active:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="activePluginsCounter">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This provides statistics about the plugin list. The total number of active plugins is normally displayed.  Other statistics may be accessed with the tooltip of this counter.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="digitCount">
        <number>4</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="BSPluginList::PluginListView" name="pluginList">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>List of available esp/esm files.</string>
     </property>
     <property name="whatsThis">
      <string>List of available esp/esm files.</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MOBase::LineEditClear" name="espFilterEdit">
       <property name="toolTip">
        <string>Filter the list of plugins.</string>
       </property>
       <property name="whatsThis">
        <string>Filter the list of plugins.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MOBase::LineEditClear</class>
   <extends>QLineEdit</extends>
   <header>lineeditclear.h</header>
  </customwidget>
  <customwidget>
   <class>BSPluginList::PluginListView</class>
   <extends>QTreeView</extends>
   <header>bspluginlist/pluginlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../modorganizer/src/resources.qrc"/>
 </resources>
 <connections/>
</ui>
